scanner/src/main/java/edu/ucr/cs/riple/scanner/out/MethodInfo.java

```diff
@@ -112,6 +113,8 @@ public String toString() {
         + "\t"
```

Comment:
Do we want to use String.format() here? I think its more efficient than concatenation of literals .

Reply:
I am not sure if using `String.format()` can lead to a measurable speed up here. Generally speaking with the current structure of the `core` and `scanner` module, with our experiments `98-99 %` of the time in the process is spent in re-building the target project :) Please let me know what you think of this and still interested in converting the concatenations to `String.format()`.

Reply:
Besides performance, `String.format()` can sometimes help with readability. However, in this particular case, I'd argue it will make this harder, not easier, to read and update this list (i.e. counting placeholders in the template string). I would consider `return String.join("\t", ...)` here, though, keeping the elements each on their own line. That would hopefully have a fast implementation of concatenation underneath, but also, is arguably more readable than what we currently have. 

Either way, I am fine with what we have in this case, just giving options :)

Reply:
I did not know about `String.join()`, thanks for noting it.

753602ebbcd8dc01c2b38b517532cf9eabcf0ac3